cedar-policy-validator/src/schema.rs

```diff
@@ -411,6 +412,7 @@ impl ValidatorSchema {
         undeclared_parent_entities: impl IntoIterator<Item = EntityType>,
         action_ids: &HashMap<EntityUID, ValidatorActionId>,
         undeclared_parent_actions: impl IntoIterator<Item = EntityUID>,
+        common_types: &HashMap<&Name, Type>,
```

Comment:
Nit: we don't need reference to hashmap keys here. Maybe change it to an iterator?